### DISCUSSION TITLE
How is the initial water content set?

### DISCUSSION THREAD

**Discussion by 1065844566** [phydrus/phydrus#49] [Q&A] [answered]
I checked all the examples with the source code and found that the initial head is given when creating the profile, and it seems impossible to set different head conditions in multiple soil layers. In the function create_profile, it seems that only the initial head can be set, but not the initial water content.

In the natural state, if the soil is significantly stratified, there will be a big difference in water content in different soil layers. Is it incorrect to default to one head content for multiple soil layers?

And clarification needs to be given that the starting position of the head is at z=0? This feature can be set to a custom one.

**Comment by yonayeku** [comment]
Follow, I have the same issue

**Comment by martinvonk** [comment, marked as answer]
The easiest solution is to calculate the initial pressure head on your own. Dependent on your soil hydraulic properties model you can calculate the pressure head from the water content. You insert your own initial pressure head in the profile DataFrame (column `h`) and add it to the model using ml.add_profile().

**Reply by zhiwenmao** [reply]
When I do not have any Measured pressure head data, and hard to measured the in-situ pressure head data(I do not have any relative 
instrument), how could I get the soil water retention curve?

**Comment by mvremec** [comment]
Hi all,
and thank you for your answers Martin! 
1. The initial conditions can be given in terms of pressure head or water content. Set `linitw=True` (in the `add_waterflow()`) if you want to give your initial conditions as water content [-]. 
2. As Martin wrote, you can add the initial pressure head/water content in the profile DataFrame. Check our examples on how to create and add the profile.

Hope this helps,
Matevz

**Reply by zhiwenmao** [reply]
Thanks a lot! I thought the package didn't have the function to set the initial soil water content before, but I found I was wrong. It has the function, I just did not get it before. I guess I know how to set it now. Thank you for sparing a time to answer the question again.

**Comment by zhiwenmao** [comment]
In the original hydrus PC-program, if you set your in-situ soil moisture as the initial condition, the data is added in the h column in PROFILE.DAT while the title ‘h’ does not change at all. so if you wanna set your in-situ soil moisture as the initial condition, you just need to modify the h column and set the model parameter linitw=True in 'add_waterflow()'.

**Reply by 1065844566** [reply]
Hi brother.
Can you share the code? Or WeChat?

**Reply by zhiwenmao** [reply]
you just need to modify your data in your PROFILE.DAT file, and add line  in the code editor like ----> profile = ps.profile.profile_from_file("PROFILE_example3.DAT", ws="profiles"),  as shown in the example3  of the Heat transport Example document by the phydrus package author.

**Reply by Dumc23** [reply]
Do you mean to input the soil moisture content directly into h?
